### DISCUSSION TITLE
Moving selection to different page - is it possible?

### DISCUSSION THREAD

**Discussion by QubaB** [saber-notes/saber#979] [Ideas]
When I select something, it is possible only to move it on the current page.

Is there a way to move selection to different page?

**Comment by jboschuk** [comment]
Plus one for this. It would be a super useful feature if you need to make more space in a note and move things to the next page.

**Comment by sammdu** [comment]
Following up in 2025 - still a problem

**Comment by sammdu** [comment]
Actually, it's tracked at #405
